src/components/DiffView/index.tsx

```diff
@@ -197,17 +204,29 @@ export class DiffViewBase extends React.Component<Props> {
         messages = selectedMessageMap.byLine[line];
       }
 
-      let widget = null;
+      let widget = (
+        <>
+          {enableCommenting && change.isInsert && (
+            <CommentList
+              addonId={version.addon.id}
+              fileName={version.selectedPath}
+              line={line || null}
```

Comment:
I guess TS wanted you to fall back to `null`? This should not be set to `null` so maybe you need to throw an error instead.

Reply:
I missed this one when I was adjusting the one below. I only have to gate this on `line` and all is fine.